app/services/HolderOfAuthenticationCode.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import utils.FileUtil
+
+case class HolderOfAuthenticationCode(value: String)
```

Comment:
Is it a holder of AUTHENTICATION or holder of AUTHORISATION - a bit confused?

Reply:
Fixed

Reply:
I think it's Holder of Authentication but inside contains Authorisation code. Just to clarify